roles/network_plugin/kube-ovn/templates/cni-kube-ovn.yml.j2

```diff
@@ -249,28 +85,32 @@ spec:
           - --node-switch-cidr={{ kube_ovn_node_switch_cidr }}{% if enable_dual_stack_networks %},{{ kube_ovn_node_switch_cidr_ipv6 }}{% endif %}{{ '' }}
           - --service-cluster-ip-range={{ kube_service_addresses }}{% if enable_dual_stack_networks %},{{ kube_service_addresses_ipv6 }}{% endif %}{{ '' }}
           - --network-type={{ kube_ovn_network_type }}
-          - --default-interface-name={{ kube_ovn_default_interface_name | default('') }}
-          - --default-vlan-id={{ kube_ovn_default_vlan_id }}
+          - --default-provider-name={{ kube_ovn_default_provider_name | default('')}}
+          - --default-interface-name={{ kube_ovn_default_vlan_interface_name | default('') }}
+          - --default-vlan-id={{ kube_ovn_default_vlan_id | default('') }}
+          - --default-vlan-name={{ kube_ovn_default_vlan_name | default('') }}
```

Comment:
Avoid the use of `default` where possible, just have the value defined in roles defaults/main.yml